blaze/json_dumps_trusted.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import absolute_import, division, print_function
+
+import types
+from collections import Callable
+
+from .dispatch import dispatch
+from functools import partial, wraps
+
+from .json_dumps import json_dumps
+
+
+json_dumps_trusted_ns = dict()
```

Comment:
how so, iterating over the dict and updating each k/v pair should be functionally equivalent? At the end of this module would the `func` maps be different?